paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormFragment.kt

```diff
@@ -163,6 +156,10 @@ internal class USBankAccountFormFragment : Fragment() {
             ViewGroup.LayoutParams.MATCH_PARENT
         )
 
+        viewModel.result.launchAndCollectIn(viewLifecycleOwner) { result ->
```

Comment:
What happens if the activity is recreated, we'd listen for this event, and `handleUSBankAccountConfirmed` again. Would that be a problem?

Reply:
This won’t happen with the current implementation, as we’re closing the bottom sheet when `handleUSBankAccountConfirmed()` is called. I understand that this is “implicit knowledge” and we might change the flow at a later time, at which point this could become a problem.

For now, I’ll turn `result` into a nullable `StateFlow`. As a follow-up, I can make the communication between the view models more direct, rather than going via the Fragment.